example/gluon/lipnet/main.py

```diff
@@ -0,0 +1,41 @@
+"""
```

Comment:
I'm wondering if this description should go below the license... I think the linters for the Apache license are looking for it to start the file.

Reply:
fixed